tests/NPSP.py

```diff
@@ -16,6 +16,7 @@
 from cumulusci.robotframework.utils import selenium_retry
 import sys
 from email.mime import text
+from _socket import timeout
```

Comment:
This import is unused and should be removed.

Reply:
I think this got added when I used timeout= 60 on my keyword "wait for locator"

Reply:
_socket.timeout is a class from the Python stdlib but you're not using that class, you're just passing timeout as the name of an argument which is set to 60. So you don't need the import.